### DISCUSSION TITLE
Manually add pictures to character groups

### DISCUSSION THREAD

**Discussion by zhimibuwudn** [immich-app/immich#12183] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The translation of your sentence into English would be:
"However, the picture does not have a frontal view of the person, manually add it to the collection of images related to the person for overall organization, review, and management."

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#7301
